Formula/as-tree.rb

```diff
@@ -0,0 +1,17 @@
+class AsTree < Formula
+  desc "Print a list of paths as a tree of paths 🌳"
+  homepage "https://github.com/jez/as-tree"
+  url "https://github.com/jez/as-tree/archive/0.12.0.tar.gz"
+  sha256 "2af03a2b200041ac5c7a20aa1cea0dcc21fb83ac9fe9a1cd63cb02adab299456"
+  license "BlueOak-1.0.0"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "as-tree", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Hi thanks, I'm not sure this applies for static binaries like rust and go, but ok!